codegenerator/integration_tests/tests/test_indexers/dynamic_contracts/config-dynamic-contracts.yaml

```diff
@@ -3,7 +3,7 @@ name: kwenta-factory-example
 field_selection:
   transaction_fields:
     - hash
-    - transaction_index
+    - transactionIndex
```

Comment:
Didn't we decide to change it to `index`? Or is it a part of another PR?

Reply:
Have a look at the thread on this closed PR https://github.com/enviodev/hypersync-client-node/pull/24

It was decided that it's best to keep it transactionIndex